### PR TITLE
[WIXMP-707] Add upload abort

### PR DESCRIPTION
**Pull request by bildja**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "media-platform-js-sdk",
-  "version": "6.0.0-alpha.3",
+  "version": "6.0.0-alpha.4",
```
Comment (maxaon) [inline package.json]: pr that changes version of package?  very strange idea

File: test/helpers/wait-a-moment.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export const waitAMoment = (timeout: number = 0): Promise<void> => {
```
Comment (maxaon) [inline test/helpers/wait-a-moment.ts]: If you use medieval moments, set timeout to 90 seconds please https://en.wikipedia.org/wiki/Moment_(time)
Or better to name it to delay, or use node lib :)

File: package.json (hunk 1 above)
Comment (maxaon) [inline package.json]: Changing version in pull requests? strange idea. What if you will have more than one open pr?

File: test/helpers/wait-a-moment.ts (hunk 2 above)
Comment (bildja) [reply test/helpers/wait-a-moment.ts]: No, it's not medieval, it's a modern understanding of the word and I like this name, it's not a first project I have this function for.
Please suggest an appropriate node lib

File: package.json (hunk 1 above)
Comment (bildja) [reply package.json]: @maxaon you are absolutely right, I will not do this in future

Comment (bildja) [reply package.json]: Okay, I will not do this and this time also

File: src/public/platform/uploader/upload-job.ts [hunk 3]
```diff
@@ -55,11 +62,11 @@ export class UploadJob extends EventEmitter {
    * @returns {UploadJob}
    */
   run(fileUploader: FileUploader): this {
-    if (this.state === 'running') {
+    if (this.state === UploadJobState.RUNNING) {
       console.warn('job already running');
```
Comment (adavidof) [inline src/public/platform/uploader/upload-job.ts]: do we need log?

Comment (bildja) [reply src/public/platform/uploader/upload-job.ts]: @adavidof  It's a warning and it was there. Moreover, there is a test which checks if the warning is logged to console

File: test/helpers/wait-a-moment.ts (hunk 2 above)
Comment (maxaon) [reply test/helpers/wait-a-moment.ts]: https://www.npmjs.com/package/delay

Comment (bildja) [reply test/helpers/wait-a-moment.ts]: @maxaon the implementation is pretty trivial (unlike schema), I'll stick with my version

Comment (adavidof) [reply test/helpers/wait-a-moment.ts]: delay is better :)

**Pull request merged**